viewer/packages/pointclouds/src/potree-three-loader/rendering/shaders/pointcloud.vert

```diff
@@ -548,7 +548,10 @@ void main() {
 	#elif defined color_type_point_index
 		vColor = indices.rgb;
 	#elif defined color_type_classification
-		vColor = classification.rgb;
+		vec3 rgb = getRGB();
+		vColor = rgb == vec3(0.0, 0.0, 0.0) 
+			? classification.rgb 
+			: mix(classification.rgb, rgb, 0.6);
```

Comment:
I guess 0.6 is mix ratio, should it be more explicit? Or maybe hold it as a variable

Reply:
I think ok for now - there's a lot of hardcoded values in there... But I changed the constant a bit to emphasise classification color.